diff -Naur spotipy-2.17.1.old/setup.py spotipy-2.17.1/setup.py
--- spotipy-2.17.1.old/setup.py	2021-03-16 13:43:45.439463410 +0100
+++ spotipy-2.17.1/setup.py	2021-03-16 13:48:42.387420257 +0100
@@ -26,9 +26,9 @@
     author_email="paul@echonest.com",
     url='https://spotipy.readthedocs.org/',
     install_requires=[
-        'requests>=2.25.0',
-        'six>=1.15.0',
-        'urllib3>=1.26.0'
+        'requests>=2.22.0',
+        'six>=1.14.0',
+        'urllib3>=1.25.7'
     ],
     tests_require=test_reqs,
     extras_require=extra_reqs,
